armi/reactor/composites.py

```diff
@@ -1564,12 +1564,18 @@ def getFissileMassEnrich(self):
             return 0.0
 
     def getUraniumNumEnrich(self):
-        """Returns U-235 number fraction."""
-        u8 = self.getNumberDensity("U238")
-        if u8 < 1e-10:
+        """Returns fissile uranium number fraction."""
+        U = elements.bySymbol["U"]
+        compNucs = set(self.getNuclides())
+        uraniumNucs = set(nuc.name for nuc in U.nuclides)
+        uraniumNucsPresent = compNucs & uraniumNucs
```

Comment:
Same simplification as the number enrichment?

Reply:
Well, if they were approximately the same speed, I would definitely vote the shorter more readable version.